packages/create-sitecore-jss/src/templates/angular/.env

```diff
@@ -0,0 +1,35 @@
+# Your Sitecore API key is needed to build the app. Typically, the API key is
+# defined in `scjssconfig.json` (as `sitecore.apiKey`). This file may not exist
+# when building locally (if you've never run `jss setup`), or when building in a
+# higher environment (since `scjssconfig.json` is ignored from source control).
+# In this case, use this environment variable to provide the value at build time.
+SITECORE_API_KEY=
+
+# Your Sitecore API hostname is needed to build the app. Typically, the API host is
+# defined in `scjssconfig.json` (as `sitecore.layoutServiceHost`). This file may
+# not exist when building locally (if you've never run `jss setup`), or when building
+# in a higher environment (since `scjssconfig.json` is ignored from source control).
+# In this case, use this environment variable to provide the value at build time.
+SITECORE_API_HOST=
+
+# Your GraphQL Edge endpoint. This is required for Sitecore Experience Edge.
+# For Sitecore XM, this is typically optional. By default, the endpoint is calculated using
+# the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`.
+GRAPH_QL_ENDPOINT=
+# Overrides `graphQLEndpointPath` defined in your `package.json`.
```

Comment:
We don't have this in Next.js .env. Do we need it? I'm not sure there would be a case this would be used if you can set `GRAPH_QL_ENDPOINT`.

Reply:
Agree, not so much cases